include/circt/Dialect/MSFT/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+##===- CMakeLists.txt - MSFT dialect build definitions --------*- cmake -*-===//
+##
+## Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+## See https://llvm.org/LICENSE.txt for license information.
+## SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+##
+##===----------------------------------------------------------------------===//
+##
+##
+##===----------------------------------------------------------------------===//
```

Comment:
nit: Plz drop the bottom block of the head comment if it doesn't contain anything.